packages/frontend/core/src/modules/multi-tab-sidebar/multi-tabs/tabs/chat.tsx

```diff
@@ -21,12 +21,16 @@ const EditorChatPanel = ({ editor }: SidebarTabProps) => {
     if (!editor) return;
     const pageService = editor.host.spec.getService('affine:page');
 
-    pageService.slots.docLinkClicked.on(() => {
-      (chatPanelRef.current as ChatPanel).doc = editor.doc;
-    });
-    pageService.slots.editorModeSwitch.on(() => {
-      (chatPanelRef.current as ChatPanel).host = editor.host;
-    });
+    const disposable = [
+      pageService.slots.docLinkClicked.on(() => {
+        (chatPanelRef.current as ChatPanel).doc = editor.doc;
+      }),
+      pageService.docsService.onModeChange(() => {
```

Comment:
`docsService` -> `docModeService`

Reply:
Thanks for your comment! That is a mistake and I will fix it.